src/iexamine.cpp

```diff
@@ -2780,6 +2783,19 @@ void iexamine::water_source(player &p, const tripoint &examp)
     g->handle_liquid( water, nullptr, 0, &examp );
 }
 
+void iexamine::milk_source( monster *source_mon )
+{
+    item milk( "milk", 0, 1 );
+    if( !source_mon->has_effect( effect_milked ) ) {
```

Comment:
Why does this case print no message?
If it is supposed to print it, you could just skip this case and handle it by one below: if a creature has no effect, `get_effect_dur` will return 0 anyway.